### ISSUE TITLE
Not compiling everytime that I add a new file

### ISSUE THREAD

**Issue by Alexisvt** [alm-tools/alm#97]
Hi @basarat! Im been playing with the editor and I dont know If Im doing something wrong.

I will try to detail the steps to reproduce this error.

1. First I initialize the project with alm -i
2. Then I add a file with the `a` command in the file tree panel.
3. Then I add some code there and try to call the command to `toggle output js file` and Im receiving this error

![image](https://cloud.githubusercontent.com/assets/7120647/15455582/cf3819a8-2015-11e6-8954-298433808049.png)

When I stop alm tool in the command window, and start it again, the autocompile options is working on the files that I added before and If I try to add a new file following the steps before is doing the same behavior.

This is how my tsconfig file looks:

```json
{
  "compilerOptions": {
    "target": "es5",
    "module": "commonjs",
    "moduleResolution": "node",
    "isolatedModules": false,
    "jsx": "react",
    "experimentalDecorators": true,
    "emitDecoratorMetadata": true,
    "declaration": false,
    "noImplicitAny": false,
    "removeComments": true,
    "noLib": false,
    "preserveConstEnums": true,
    "suppressImplicitAnyIndexErrors": true
  },
  "compileOnSave": true,
  "exclude": [
    "node_modules",
    "typings/browser",
    "typings/browser.d.ts"
  ]
}
```

**Comment by Alexisvt** [comment]
Hi @basarat I discovered how to make it work, again I don't know if this behavior is normal (in the case that it is i think that may be confusing!) .

The workaround is to add a new file normally as I described before and then press `shift + F6` to `sync` the project, and after that I'm receiving intellisense and my file is compiled!. See the image:

![almeditor](https://cloud.githubusercontent.com/assets/7120647/15455652/f4b5502c-2017-11e6-9317-02887111b2ee.gif)

What I expect is to get that behavior automatically when I add a new file; as a user I'm not expecting to press `shift + F6` to make it work! 

Again thanks for this wonderful project :smile:

**Comment by basarat** [comment]
Sync should definitely not be required for such a common clase. I'll try and fix it today.

# Background

I do have some automatic detection of *is sync required* [here](https://github.com/alm-tools/alm/blob/3f4ec7a3fd461550e394f027161af18694ab6527/src/server/workers/lang/activeProject.ts#L61-L97). I think it failed to match the file in this case. Again, I'll have a look after work today :rose:

**Comment by basarat** [comment]
Released with `0.0.59` :rose:
